locale/en-US/add-on.properties

```diff
@@ -170,12 +134,6 @@ legal_text_privacy=Privacy Notice
 powered_by_beforeLogo=Powered by
 powered_by_afterLogo=
 
-## LOCALIZATION_NOTE (feedback_rejoin_button): Displayed on the feedback form after
-## a signed-in to signed-in user call.
-feedback_rejoin_button=Rejoin
-## LOCALIZATION NOTE (feedback_report_user_button): Used to report a user in the case of
-## an abusive user.
-feedback_report_user_button=Report User
 feedback_window_heading=How was your conversation?
 feedback_request_button=Leave Feedback
 
```

Comment:
While we're removing from add-on, `infobar_screenshare_browser_message2` `login_expired` `service_not_available` `rooms_panel_title` `rooms_signout_alert` additionally seem to be unused. Some added by various land-strings-early commits: https://github.com/mozilla/gecko-dev/commit/6f8791f0 and https://github.com/mozilla/gecko-dev/commit/d424f255